docassemble/VTFeeWaiver/data/questions/VT_fee_waiver.yml

```diff
@@ -2215,9 +2218,9 @@ attachments:
       - "users1_income_monthly_total": |
           % if defined("users1_income_self_employment_monthly_amount"):
           ${ currency(float(jobs.total(source=["full time","part time"],times_per_year=12) + other_incomes.total(times_per_year=12)) + users1_income_self_employment_monthly_amount,symbol="") }
-          % elif employed:
+          % elif jobs.total(source=["full time","part time"],times_per_year=12) > 0:
```

Comment:
This won't include people if they only have a self-employed job.  It could be changed to `source=["full time","part time","self employed"]`